docs/api/messages.md

```diff
@@ -17,4 +17,16 @@ graph RL
     ModelResponse("ModelResponse(parts=list[...])") --- ModelMessage("ModelMessage<br>(Union)")
 ```
 
+## Citations
```

Comment:
We don't need this here; for API docs purposes this context can be on docstrings.